src/classic/element/__tests__/ReactElementValidator-test.js

```diff
@@ -135,7 +141,7 @@ describe('ReactElementValidator', function() {
     spyOn(console, 'warn');
     var Component = React.createFactory(ComponentClass);
 
-    Component(null, { 1: Component(), 2: Component() });
+    Component(null, frag({ 1: Component(), 2: Component() }));
```

Comment:
I foresee more of this pattern in the future since `ReactFragment.create()` is a little verbose. Maybe package this as `React.addons.frag`?